presentation-compiler/src/main/dotty/tools/pc/MetalsInteractive.scala

```diff
@@ -110,67 +114,80 @@ object MetalsInteractive:
       pos: SourcePosition,
       indexed: IndexedContext,
       skipCheckOnName: Boolean = false
-  ): List[(Symbol, Type)] =
+  ): List[(Symbol, Type, Option[String])] =
     import indexed.ctx
     path match
+      // Handle select on named tuples
+      case (Apply(Apply(TypeApply(fun, List(t1, t2)), List(ddef)), List(Literal(Constant(i: Int))))) :: _
```

Comment:
Should we move it a bit down? Since it's rather complex case maybe it would more efficient to have more obvious case first

Reply:
Okay, though the "default" one is a bit of a "catch all" so it has to be later anyway.